lib/clientHelpers.ts

```diff
@@ -13,6 +13,10 @@ import {
 import { Submission } from "@lib/types/submission-types";
```

Comment:
Renamed helpers.ts to clientHelpers.ts to help make it clear that client util functions can go here. The rest of the lib directory is for server related util functions